src/definitions/behaviors/form.js

```diff
@@ -1403,7 +1412,7 @@
                                 settings.onValid.call($field);
                             }
                         } else {
-                            if (showErrors) {
+                            if (showErrors && newIdentifier) {
```

Comment:
See above. The extra check isn't necessary as this "else" branch will only be called when `fieldValid` is false which is already set when at least one element was detected as invalid